t/cli/test_cmd.sh

```diff
@@ -24,9 +24,9 @@ git checkout conf/config.yaml
 # remove stale conf server sock
 touch conf/config_listen.sock
 ./bin/apisix start
-sleep 0.5
```

Comment:
Why do you sleep so long?

Reply:
The tests were failing intermittently on CI as well as my machine. They passed only after giving a long enough sleep between start and stop.

Reply:
I don't think it takes that long, our ci test is too slow now, please reduce the time

Reply:
Okay Im bringing it to 2.5 but I have seen this test fail a lot due to less sleep time so I hope 2.5 is not too slow, not too fast.